chunk/tag.go

```diff
@@ -45,7 +47,7 @@ const (
 type Tag struct {
 	Uid       uint32    // a unique identifier for this tag
 	Name      string    // a name tag for this tag
-	Address   Address   // the associated swarm hash for this tag
+	Address   string    // the associated swarm hash for this tag
```

Comment:
why this change?

Reply:
Misunderstood the comment. changing it back to Address.